configure.ac

```diff
@@ -345,6 +345,9 @@ if test x$target_win32 = xyes; then
    AC_DEFINE(TARGET_WIN32, 1, [Target Platform is Win32])
 fi
 
+# Defined for all targets/platforms using classic Windows API support.
+AC_DEFINE(HAVE_CLASSIC_WINAPI_SUPPORT, 1, [Use classic Windows API support])
```

Comment:
I believe we should define the other `HAVE_<family>_WINAP_SUPPORT` defines to 0 here as well.